misc/test-lib/helpers.js

```diff
@@ -1,25 +1,33 @@
 // jasmine matcher for expecting an element to have a css class
 // https://github.com/angular/angular.js/blob/master/test/matchers.js
 beforeEach(function() {
-  this.addMatchers({
-    toHaveClass: function(cls) {
-      this.message = function() {
-        return "Expected '" + this.actual + "'" + (this.isNot ? ' not ' : ' ') + "to have class '" + cls + "'.";
-      };
-
-      return this.actual.hasClass(cls);
+  jasmine.addMatchers({
+    toHaveClass: function(util, customEqualityTesters) {
+      return {
+        compare: function(actual, expected) {
+          return {
+            pass: actual.hasClass(expected)
+          };
+        }
+      }
     },
-    toBeHidden: function () {
-      var element = angular.element(this.actual);
-      return element.hasClass('ng-hide') ||
-        element.css('display') == 'none';
+    toBeHidden: function(util, customEqualityTesters) {
+      return {
+        compare: function(actual, expected) {
+          return {
+            pass: actual.hasClass('ng-hide') || actual.css('display') === 'none'
+          };
+        }
+      }
     },
-    toHaveFocus: function () {
-      this.message = function () {
-        return 'Expected \'' + angular.mock.dump(this.actual) + '\' to have focus';
-      };
-
-      return document.activeElement === this.actual[0];
+    toHaveFocus: function(util, customEqualityTesters) {
+      return {
+        compare: function(actual, expected) {
+          return {
+            pass: util.equals(document.activeElement, actual[0], customEqualityTesters)
```

Comment:
This is a reference check so we should probably use `===` instead of the utils equals.